index.ts

```diff
@@ -0,0 +1,28 @@
+import { IncomingMessage, ServerResponse } from 'http';
+
+const DEFAULT_POLICY = 'same-origin';
+
+type Policy = 'same-origin' | 'same-site';
+
+interface CrossOriginResourcePolicyOptions {
+  policy?: Policy;
```

Comment:
Will this typing create issues for folks? For example, I could imagine someone loading their policy from a JSON file or environment variable, which would come in as a `string`, not a `Policy`.

Reply:
Good point, I'll change to `string` for now. I think this will be less of an issue if we move to ES-style import/exports in the future, as then we can export the `Policy` type as well:

```ts
import crossOriginResourcePolicy, { Policy } from 'cross-origin-resource-policy';

const policy: Policy = process.env.POLICY;
crossOriginResourcePolicy({ policy });
```